source/style-guide/markup/format/substitutions.txt

```diff
@@ -2,13 +2,19 @@
 Substitutions
 =============
 
+In Snooty, you can use variables to represent a text value in the
+rendered form of the documentation. These variables are called
+source constants. When used consistently, they help prevent inconsistencies in
+formatting the term and make it easy to change the value for all its
+occurrences.
```

Comment:
[suggestion]

To help contextualize this, it might help to briefly mention a use case for substitutions. Consider adding something like:

"For example, you can use a substitution for the name of a MongoDB product, like Compass. If the product name changes, you only need to update the variable value to update all instances of "Compass" in the entire documentation site."